app/components/LeftPaneEntries/index.tsx

```diff
@@ -256,6 +322,7 @@ function LeftPane(props: Props) {
                 entryType: 'IMAGE',
                 lead: leadId,
                 imageRaw: capturedImageUrl,
+                leadAttachment: leadAttachmentId,
```

Comment:
i don't think this is necessary, consult with @tnagorra .

Reply:
We need this because the API excepts leadAttachment to create entryAttachment later.